config/locales/cy.yml

```diff
@@ -26,6 +26,13 @@ cy:
       href: https://llyw.cymru/
       alt: Llywodraeth Cymru
       text: Hafan
+  govwales_breadcrumbs:
+    start_again_text: Start again
```

Comment:
Might be worth adding a `TODO:` comment here to remind us to add the Welsh translations?

Reply:
We got the same in the Welsh 'CY' version of the config file.

Reply:
Yeah could even just add CY like we have done else where just to distinguish the two to make sure it is being translated?

Reply:
These will be addressed during translation.